### PR TITLE
Upstream master drwn 252

### PR DESCRIPTION
**Pull request by bmaxime-c**
Add ability to send custom config when calling to AddSwaggerGen at Application Startup
Add PATCH method for restClient feature.

### CODE REVIEW

**Comment by devpro** [comment]
@bmaxime-c Would you be able to look at the CI issue?

**Comment by bmaxime-c** [comment]
Sorry for that, a last-minute refactoring. I fixed it.

**Comment by b3nNn** [comment]
Careful @bmaxime-c , this is not the code which was supposed to be commit. We've done something wrong, this code **should not** be merged as is.

File: test/AspNetCoreApiSample.IntegrationTests/Resources/TaskResourceTest.cs [hunk 1]
```diff
@@ -93,7 +93,8 @@ public async Task AspNetCoreApiSampleTaskResourceUpdate_WhenResourceDoesNotExist
         }
 
         [Fact]
-        public async Task AspNetCoreApiSampleTaskResourceUpdate_WhenResourceDoesNotExistAndNewTitleIsNull_ReturnsHttpBadRequest()
+        public async Task
+            AspNetCoreApiSampleTaskResourceUpdate_WhenResourceDoesNotExistAndNewTitleIsNull_ReturnsHttpBadRequest()
```
Comment (devpro) [inline test/AspNetCoreApiSample.IntegrationTests/Resources/TaskResourceTest.cs:97]: I don't think this change (line break) is needed for the PR

**Review by devpro — CHANGES_REQUESTED** [review]

File: test/AspNetCoreApiSample.IntegrationTests/Resources/TaskResourceTest.cs (hunk 1 above)
Comment (bmaxime-c) [reply test/AspNetCoreApiSample.IntegrationTests/Resources/TaskResourceTest.cs:97]: Indeed, this line break was introduced by the code formatting. I can remove it.

**Comment by devpro** [comment]
You just have to increment minor version number in `Directory.Build.props` and I'll merge this PR.

**Comment by bmaxime-c** [comment]
I just incremented the version number.

**Pull request merged**